### DISCUSSION TITLE
Install Laravel 9 --dev with laravel / jetstream package | Laravel 9  release - January 25th, 2022

### DISCUSSION THREAD

**Discussion by mijelidev** [livewire/livewire#4542] [7. Bugs (archived)] [answered]
Install Laravel 9 --dev with laravel / jetstream package:

Livewire / livewire package installation failed.

 Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - livewire/livewire[v2.5.0, ..., v2.9.0] require symfony/http-kernel ^5.0 -> found symfony/http-kernel[v5.0.0-BETA1, ..., 5.4.x-dev] but the package is fixed to v6.0.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires livewire/livewire ^2.5 -> satisfiable by livewire/livewire[v2.5.0, ..., v2.9.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by joshhanley** [comment, marked as answer]
@mijelidev there is already an open discussion regarding Laravel 9 support, please see it here #4536.

Thanks!

**Reply by mijelidev** [reply]
Thanks
